assets/javascript/app.js

```diff
@@ -0,0 +1,69 @@
+var questions = [
+    {
+        q: "Which one is the best hamburger?",
+        a: ["In n Out", "Five Guys", "Jack in the box", "KFC"],
+        correct: "Five Guys"
+    },
+
+    {
+        q: "Which city is the best city in California?",
+        a: ["LA", "Santa Barbara", "San Francisco", "Irvine"],
+        correct: "San Francisco"
+    },
+    
+    {
+        q: "Which video game below is the most addictive game?",
+        a: ["Overwatch", "BattleField", "LOL", "CS:GO"],
+        correct: "Overwatch"
+    },
+
+    {
+        q: "Which movie below is the most watched movie?",
+        a: ["Titanic", "The Lion King", "Superman", "Batman"],
+        correct: "Titanic"
+    }
```

Comment:
You forgot a comma here so the console is giving a syntax error. Please make sure you have your google chrome inspect open when you are developing.